pdc/apps/unreleasedvariant/tests.py

```diff
@@ -0,0 +1,331 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is the same exact `tests.py` that was at `pdc/apps/module/tests.py`. The only difference is the last test function at the bottom.